nautobot/docs/user-guide/platform-functionality/savedview.md

```diff
@@ -43,3 +44,123 @@ The `config` attribute holds the configuration of the list view, stored as a `di
     }
 }
 ```
+
+Below is a user guide on how to create, update, save and use saved views in Nautobot.
```

Comment:
Nit - I don't think we have anything written so formal as a "documentation style guide" but personally I like to use Title Case (`Saved Views`) any time we're talking about a data model.